docs/changelog/135051.yaml

```diff
@@ -3,3 +3,13 @@ summary: Ban Limit + `MvExpand` before remote Enrich
 area: ES|QL
 type: bug
 issues: []
+highlight:
+  title: Mark Limit + MvExpand before remote Enrich as unsupported
+  body: |-
+    Due to the distribured nature of remote `ENRICH`, using a `LIMIT` command followed by a `MV_EXPAND` command before a remote `ENRICH` command
+    can lead to incorrect results. Such queries are identified by the engine and marked as unsupported, leading to query failure. Example:
+    [source,yaml]
+    ----------------------------
+    FROM *:events | SORT @timestamp | LIMIT 2 | MV_EXPAND ip | ENRICH _remote:clientip_policy ON ip
+    ----------------------------
+    Such query will now produce an error. The users are advised to rewrite it (e.g. by placing `ENRICH` earlier in the query) to avoid the error.
```

Comment:
```suggestion
  body: |-
    Queries using LIMIT followed by MV_EXPAND before a remote ENRICH can produce incorrect results due to distributed execution semantics.
    These queries are now unsupported and produce an error. Example:
    [source,yaml]
    ----------------------------
    FROM *:events | SORT @timestamp | LIMIT 2 | MV_EXPAND ip | ENRICH _remote:clientip_policy ON ip
    ----------------------------
    To avoid this error, reorder your query, for example by moving ENRICH earlier in the pipeline.
```

Reply:
Suggestion to rewrite for clarity. Maybe we should have an example of the error too?